vue-frontend/src/components/android-app-development/BlogSection.vue

```diff
@@ -0,0 +1,236 @@
+        <swiper
+          :slidesPerView="2"
+          :spaceBetween="20"
+          :breakpoints="{
+            '570': {
+              spaceBetween: 0,
+            },
+          }"
+          :centeredSlides="true"
+          class="tw-p-[1rem]"
+        >
+          <swiper-slide
+            v-for="(blog, index) in blogs"
+            :key="index"
+            class="tw-relative tw-flex tw-flex-col tw-pl-[1rem] sm:tw-pl-[2rem]"
+          >
+            <div class="tw-overflow-hidden">
+              <div class="tw-h-full tw-w-full tw-cursor-pointer">
+                <img
+                  @click="openBlog(blog.link)"
+                  :src="blog.image[0]"
+                  :srcset="`${blog.image[0]} 400w, ${blog.image[1]} 800w`"
+                  :alt="blog.title"
+                  class="tw-h-full tw-w-full tw-rounded-[10px] tw-object-cover"
+                  loading="lazy"
+                />
+              </div>
+              <div class="title tw-hidden">
+                <p
+                  class="tw-mt-2 tw-pl-2 tw-font-inter-regular tw-text-[0.875rem] tw-leading-[1.3125rem] tw-text-black-core/[0.6]"
+                >
+                  {{ blog.publishDate }}
+                </p>
+                <a
+                  @click.native="mixpanel.track('tap_android__app_blog_title')"
+                  :href="blog.link"
+                  target="_blank"
+                >
+                  <p
+                    class="tw-mt-2 tw-cursor-pointer tw-pl-2 tw-font-inter-semibold tw-text-[1.125rem] tw-leading-[1.6875rem] tw-text-black-core/[0.87]"
+                  >
+                    {{ blog.title }}
+                  </p>
+                </a>
+              </div>
+            </div>
+          </swiper-slide>
+        </swiper>
+      </div>
+      <div
+        class="tw-absolute tw-inset-0 tw-z-[0] tw-mx-auto tw-h-[430px] tw-w-[315px] tw-rounded-[10px] tw-border tw-border-black-core/[0.60] sm:tw-left-[2rem] sm:tw-w-[365px]"
+      ></div>
+    </div>
+
+    <!-- Mobile UI end -->
+    <!-- Desktop UI start -->
+    <div
+      class="tw-container tw-flex tw-hidden tw-flex-col md:tw-block md:tw-min-h-[580px] lg:tw-py-0"
+    >
+      <p
+        class="tw-mb-8 tw-text-center tw-font-inter-bold tw-text-[3.4375rem] tw-leading-[5.15625rem] tw-text-black-core/[0.87]"
+      >
+        Our Blogs
+      </p>
+      <div class="tw-flex tw-flex-row tw-space-x-5 tw-space-y-1">
+        <div
+          v-for="blog in blogs.slice(0, 1)"
+          :key="blog"
+          @click.native="mixpanel.track('tap_android_app_blog_post')"
+          class="tw-mt-[1.5rem] tw-flex tw-basis-[65%] tw-cursor-pointer tw-flex-col tw-rounded-[20px]"
+        >
+          <img
+            @click="openBlog(blog.link)"
+            :src="blog.image[1]"
+            :srcset="`${blog.image[1]} 800w`"
+            class="tw-object-fit tw-h-full tw-w-full tw-cursor-pointer tw-rounded-t-lg"
+            loading="lazy"
+            :alt="blog.title"
```

Comment:
add animation on hover in desktop design. refer [(https://www.figma.com/proto/xl4quhaEjFgUrcoMqSJn6u/Canopas-Website?type=design&node-id=19105-31711&t=ck8V7zQo24vp0zMo-0&scaling=min-zoom&page-id=8267%3A1473&starting-point-node-id=8500%3A1768)](url)